src/node_perf.cc

```diff
@@ -58,20 +54,9 @@ void performance_state::Mark(enum PerformanceMilestone milestone,
 }
 
 double GetCurrentTimeInMicroseconds() {
-#ifdef _WIN32
-// The difference between the Unix Epoch and the Windows Epoch in 100-ns ticks.
-#define TICKS_TO_UNIX_EPOCH 116444736000000000LL
-  FILETIME ft;
-  GetSystemTimeAsFileTime(&ft);
-  uint64_t filetime_int = static_cast<uint64_t>(ft.dwHighDateTime) << 32 |
-                          ft.dwLowDateTime;
-  // FILETIME is measured in terms of 100 ns. Convert that to 1 us (1000 ns).
-  return (filetime_int - TICKS_TO_UNIX_EPOCH) / 10.;
-#else
-  struct timeval tp;
-  gettimeofday(&tp, nullptr);
-  return MICROS_PER_SEC * tp.tv_sec + tp.tv_usec;
-#endif
+  uv_timeval_t tv;
+  CHECK_EQ(uv_gettimeofday(&tv), 0);
```

Comment:
We predominantly seem to write such calls as `CHECK_EQ(0, uv_foo(...))` (44 vs 11.)